Extension/src/LanguageServer/configurations.ts

```diff
@@ -988,7 +988,7 @@ export class CppProperties {
             } else {
                 // However, if compileCommands are used and compilerPath is explicitly set, it's still necessary to resolve variables in it.
                 if (configuration.compilerPath === "${default}") {
-                    configuration.compilerPath = settings.defaultCompilerPath;
+                    configuration.compilerPath = settings.defaultCompilerPath ?? undefined;
```

Comment:
I think this should be changed back -- it caused the code below to not execute when the defaultCompilerPath is null.